### PR TITLE
SDK-1294: moved packages to required

### PR DESCRIPTION
**Pull request by vol4onok**
## Issue ticket link
Ticket:https://spryker.atlassian.net/browse/SDK-1294.

## Checklist before requesting a review
- [ ] I have added ticket link to the PR description.
- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

**Review by sergeyspryker — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -33,8 +33,10 @@
         "rector/rector": "^0.13.0",
         "spryker-sdk/acp": "^0.2.0",
         "spryker-sdk/async-api": "^0.2.3",
+        "spryker-sdk/brancho": "1.0.x-dev",
```
Comment (dereuromark) [inline composer.json:36]: Doesnt look like it is ready to be a stable dependency here IMO
See also https://github.com/spryker-sdk/brancho/blob/master/.github/workflows/ci.yml#L15

Comment (vol4onok) [reply composer.json:36]: branch support >=7.3 https://github.com/spryker-sdk/brancho/blob/master/composer.json#L7

Comment (dereuromark) [reply composer.json:36]: Not a single release, untested for 8.1/8.2, quite dormant without any activity.
Sounds a bit risky, no?
Whats the reason this cannot stay optional for now?

Comment (vol4onok) [reply composer.json:36]: we have tasks for it and install composer with dev-require for prod

Comment (vol4onok) [reply composer.json:36]: I would remove it. anyway we are going to have VSC tasks. @pavelmaksimov25 what do you think?

Comment (dereuromark) [reply composer.json:36]: Those are dev tools right? IMO they should not be part of prod setup anyway.

Comment (vol4onok) [reply composer.json:36]: They were the dev tool parts. but we use them for prod. That's why I moved them to required section

Comment (dereuromark) [reply composer.json:36]: I think then you need to get the dependency (brancho) first up to speed before you officially release this
As you should then add a tag here then for merge (^0.1.0).

Comment (pavelmaksimov25) [reply composer.json:36]: SDK is a tool aggregator and it uses `brancho` in order to provide tool this for the developers. It is used in our production code. In the ticked, I've asked to speak to the tool owner first and then decide to remove this tool completely or move it to the `require` section. 
VCS task can be used for such cases but the original idea was different from what `brancho` does.
As long as SDK is not stable I don't mind having it as a prod dependency. So far it blocks us, and we're not able to install sdk with `--no-dev` flag.

Comment (dereuromark) [reply composer.json:36]: Ok

**Review by Kyrylo-Meretskyi — APPROVED** [review]

**Review by pavelmaksimov25 — APPROVED** [review]

**Review by pavelmaksimov25 — APPROVED** [review]

**Pull request merged**